corehq/apps/userreports/templates/userreports/base_report_builder.html

```diff
@@ -25,9 +25,15 @@ <h2>
         </p>
         <p class="alert alert-info">
           <i class="fa fa-info-circle"></i>
-          {% blocktrans %}
-            For even more reports please contact our team at <strong>{{ support_email }}</strong>.
-          {% endblocktrans %}
+          {% if is_saas_environment %}
+            {% blocktrans %}
+              For even more reports please contact our team at <a href="mailto:sales@dimagi.com">sales@dimagi.com</a>.
```

Comment:
How about using `settings.REPORT_BUILDER_ADD_ON_EMAIL` to avoid hard-coding emails?